source/guide_simpleid.rst

```diff
@@ -0,0 +1,229 @@
+ [isabell@stardust ~]$
+
+Before editing this file, we need to create a password/salt pair.
+
+There are several ways to go, using the MD5, SHA1 and SHA256 algorithm and an optional salt. In this example we will use the SHA256 algorithm with a random salt.
+
+First we'll generate a random salt to make this secure as possible. You are free to use any string as salt.
+
+::
+
+ [isabell@stardust ~]$ date +%s | sha256sum | base64 | head -c 16 ; echo
+ MySuperSecretSalt
+ [isabell@stardust ~]$
+ 
+With this generated hash and your password in mind we can create the SHA256 hash with 100,000 iterations for the identity file.
+
+.. warning:: Replace ``<password>`` with your password, and ``<salt>`` with the generated salt or the one you decided to chose.
+
+::
+
+ [isabell@stardust ~]$ php -r 'echo hash_pbkdf2("sha256", "<password>", "<salt>", 100000).PHP_EOL;'
+ [isabell@stardust ~]$
+
+With our example values this would be
+
+::
+
+ [isabell@stardust ~]$ php -r 'echo hash_pbkdf2("sha256", "MySuperSecretPassword", "MySuperSecretSalt", 100000).PHP_EOL;'
+ 5fd924625f6ab16a19cc9807c7c506ae1813490e4ba675f843d5a10e0baacdb8
+ [isabell@stardust ~]$
+
+Then open the identity file you copied before to /var/www/virtual/$USER/simpleid/identities/<username>.identity in your favorite editor.
+
+Search for the pass line and edit it as described.
+
+.. code-block:: php
+
+ pass="<hash>:pbkdf2:sha256:<salt>"
+
+In our example we would use this passline:
+
+.. code-block:: php
+
+ pass="5fd924625f6ab16a19cc9807c7c506ae1813490e4ba675f843d5a10e0baacdb8:pbkdf2:sha256:MySuperSecretSalt"
+ 
+Since this is our first user, it should be set as administrator by changing
+
+.. code-block:: php
+
+ ;administrator=1
+ 
+to (remove the ``;``)
+
+.. code-block:: php
+
+ administrator=1
+ 
+The last thing is to set an `identifier <http://simpleid.koinic.net/docs/1/identity-requirements/#identifier>`_ using an unique URL.
+
+.. warning:: Replace ``<username>`` with your Uberspace username!
+
+.. code-block:: php
+
+ identity="https://<username>.uber.space/openid/"
+
+This will be the URL you'll have to use for registering with websites. We'll create the folder and fill it in a later step. In this example we will use:
+
+.. code-block:: php
+
+ identity="https://isabell.uber.space/openid/"
+
+.. warning:: Don't forget the closing ``/`` or you'll get an error if you try to use the identity.
+
+After saving the file, you should be able login for the first time at your SimpleID installation located at the defined Base URL ``https://<username>.uber.space/simpleid``.
+
+Create identifier
+-----------------
+
+For finally using this identity you'll need to create a file at the URL we've defined in the .identity file.
+
+::
+
+ [isabell@stardust ~]$ mkdir ~/html/openid
+ [isabell@stardust ~]$
+
+Create the file index.htm with the following content:
```

Comment:
That should be "Create the file ~/html/openid/index.htm with the following content:"